.github/workflows/generate-checksums.yml

```diff
@@ -178,13 +72,13 @@ jobs:
       - name: Download Checksums
         uses: actions/download-artifact@v4
         with:
-          name: ${{ needs.generate-checksum.outputs.artifact-name }}
-          path: ${{ env.OUTPUT_LOCAL_LOCATION }}
+          name: ${{ needs.repro-ci.outputs.artifact-name }}
+          path: ${{ env.ARTIFACT_LOCAL_LOCATION }}
 
       - name: Move Checksums to Repo
         run: |
           mkdir -p ${{ inputs.committed-checksum-location }}
-          mv ${{ env.OUTPUT_LOCAL_LOCATION }}/checksum/* ${{ inputs.committed-checksum-location }}
+          cp --recursive --verbose ${{ env.ARTIFACT_LOCAL_LOCATION }}/*/* ${{ inputs.committed-checksum-location }}
```

Comment:
This has switched from `mv` to `cp --recursive` and including everything in a sub-directory of `env.ARTIFACT_LOCAL_LOCATION` rather than just under the `checksum` sub-dir.

Can you explain those changes so we have a record of why? Thanks!

Reply:
This will support the case where we have potentially other subdirectories that we want to copy, like `additional-artifact-content-paths`, or maybe `performance` or something like that.